plugins/cinema4d/import_cast.pyp

```diff
@@ -366,21 +374,21 @@ def importSkeletonIKNode(doc, modelNull, skeleton, boneIndexes):
         ikTag[c4d.ID_CA_IK_TAG_SOLVER] = 2
         ikTag[c4d.ID_CA_IK_TAG_TIP] = endBone
         ikTag[c4d.ID_CA_IK_TAG_TARGET] = ikTargetNull
-        
+
         poleVectorBone = handle.PoleVectorBone()
         if poleVectorBone is not None:
             poleVector = boneIndexes[poleVectorBone.Hash()]
-            ikPoleNull = BaseObject(c4d.Onull)
-            ikPoleNull.SetName(poleVector.GetName() + "_Pole")
-            ikPoleNull.SetMg(poleVector.GetMg())
-            ikPoleNull.SetAbsRot(Vector(0, 0, 0))
-            
-            doc.InsertObject(ikPoleNull, startBone)
-            
             ikTag[c4d.ID_CA_IK_TAG_POLE] = poleVector
+            ikTag[c4d.ID_CA_IK_TAG_POLE_AXIS] = POLE_AXIS_X
+            ikTag[c4d.ID_CA_IK_TAG_POLE_TWIST] = poleVector[c4d.ID_BASEOBJECT_REL_ROTATION,c4d.VECTOR_X]
 
+        poleBone = handle.PoleBone()
+        if poleBone is not None:
```

Comment:
`if handle.PoleBone():`

Reply:
It definitely makes more sense, I just tried to copy your plugins to have it easier to maintain. Your Blender and Maya addon use the
```
poleBone = handle.PoleBone()
        if poleBone is not None:
```
but I'll remove it!

Reply:
I think it looks a bit better with the poleBone.Name() in the warning in my opinion.